usage/dist/src/main/dist/conf/brooklyn/default.catalog.bom

```diff
@@ -0,0 +1,41 @@
+
+# this catalog bom is an illustration supplying a few useful sample items
+# and templates to get started using Brooklyn
+
+brooklyn.catalog:
+  version: 0.7.0-SNAPSHOT  # BROOKLYN_VERSION
+  items:
+
+  # load everything in the classpath with a @Catalog annotation
+  - scanJavaAnnotations: true
+
+  - id: server
+    description: |
+      Provision a server, with customizable provisioning.properties and credentials installed, 
+      but no other special software process or scripts executed.
+    item:
+      type: brooklyn.entity.basic.EmptySoftwareProcess
```

Comment:
I'd prefer if we have consistent structure, rather than demonstrating different possible structures within the same file (that will just confuse users, as to whether they mean the same thing).

Suggest we include `services: `, as is done in `server-template`.